lib/codegen/src/verifier/mod.rs

```diff
@@ -186,6 +186,12 @@ impl<'a> Verifier<'a> {
 
                 cur = base;
             }
+            
+            if let ir::GlobalValueData::VMContext { .. } = self.func.global_values[cur] {
+                if self.func.special_param(ir::ArgumentPurpose::VMContext).is_none() {
```

Comment:
CI complains that this line should be formatted differently. You can run `./format-all.sh` from Cretonne's top level dir to have the expected formatting done for you.